### PR TITLE
Fix: Error when plotting polar data using contour

### PR DESCRIPTION
**Pull request by r5r3**
Change in path_to_geos: for polygons with `Path.CLOSEPOLY` one more segment is required.

### CODE REVIEW

**Comment by pelson** [comment]
Closes https://github.com/SciTools/cartopy/issues/791.

File: docs/source/contributors.rst [hunk 1]
```diff
@@ -36,6 +36,7 @@ the package wouldn't be as rich or diverse as it is today:
  * Daniel Eriksson
  * Crispian Batstone
  * Daryl Herzmann
+ * Robert Redl
```
Comment (pp-mo) [inline docs/source/contributors.rst]: Does this really belong ??

File: lib/cartopy/mpl/patch.py [hunk 2]
```diff
@@ -185,13 +185,10 @@ def path_to_geos(path, force_ccw=False):
...
         if all(verts_same_as_first):
             geom = sgeom.Point(path_verts[0, :])
-        elif (path_verts.shape[0] > 3 and
-                (path_codes[-1] == Path.CLOSEPOLY or
-                 verts_same_as_first[-1])):
-            if path_codes[-1] == Path.CLOSEPOLY:
-                geom = sgeom.Polygon(path_verts[:-1, :])
-            else:
-                geom = sgeom.Polygon(path_verts)
+        elif path_verts.shape[0] > 4 and path_codes[-1] == Path.CLOSEPOLY:
```
Comment (pp-mo) [inline lib/cartopy/mpl/patch.py]: :+1:

File: docs/source/contributors.rst (hunk 1 above)
Comment (pelson) [reply docs/source/contributors.rst]: Sure does: ``If you have been excluded from the list, please add yourself and submit a pull request.``

Comment (pp-mo) [reply docs/source/contributors.rst]: Sorry, hadn't noticed who was proposing this !

**Comment by pelson** [comment]
Thanks for this @r5r3! Even though the CI is failing I'm going to merge and move forwards (I'll fix any fallout on master).

Thanks again! Good fix. 👍

**Comment by pelson** [comment]
Actually. I've noticed we don't have a CLA for your contribution @r5r3. Would you mind sending one over so that we can merge this. CLA described at http://scitools.org.uk/governance.html#contributors.

**Comment by QuLogic** [comment]
Any updates here?

**Comment by r5r3** [comment]
Sorry for the delay. I just signed and submitted the CLA.

**Comment by pelson** [comment]
This PR is good to merge IMO. @r5r3 your user isn't associated to the commit in this PR, hence the CLA check failing. Would you mind modifying your commit to include your (no-reply) email address as discussed in https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user. As you will be modifying the commit, would you also mind rebasing ``git rebase upstream/master`` to get the tests passing please?

Thanks!

**Comment by r5r3** [comment]
The commit should now be associated to my account.

**Comment by pelson** [comment]
Thanks @r5r3. Merged in #1021.

**Pull request closed without merging**